share/mdnjs/parse.sh

```diff
@@ -1,5 +1,15 @@
 #!/bin/bash
-python parse.py --out output.txt \
+rm output.txt
+
+python parse.py --out output.api.txt \
```

Comment:
@jdorweiler can you please tell me how are these two calls different? The only thing that seem to have changed is the `cachedir` and the `output` file. Does that mean that some functionality depends on those two variables? Did I miss something?